mbf_msgs/action/ExePath.action

```diff
@@ -57,3 +57,4 @@ float32 dist_to_goal
 float32 angle_to_goal
 geometry_msgs/PoseStamped  current_pose
 geometry_msgs/TwistStamped last_cmd_vel  # last command calculated by the controller
+geometry_msgs/TwistStamped feedback_cmd_vel  # robot actual velocity feedback from wheel encoder
```

Comment:
I don't like `feedback`... what about `current`? or `actual`?

Reply:
What about `odom_vel_`?

Reply:
I prefer odom_vel_ too. but I think ExePath action's feedback is for user to monitor each mobile robot behavior, so maybe follow whole naming rule, call actual_vel_ ?

Reply:
`current_vel`, and move above last_cmd_vel;
mostly to match `current_pose`; both are physical properties:
```
geometry_msgs/PoseStamped  current_pose
geometry_msgs/TwistStamped current_vel   # robot actual velocity from the odometry
geometry_msgs/TwistStamped last_cmd_vel  # last command calculated by the controller

Reply:
Jap, this line should be removed here. So we can not change the messages anymore. For noetic I think this will be ok, if we hurry a bit, but not for melodic and before.